thirdparty/kyaml/runfn/runfn.go

```diff
@@ -35,21 +35,13 @@ type RunFns struct {
 	// Path is the path to the directory containing functions
 	Path string
 
-	// FunctionPaths Paths allows functions to be specified outside the configuration
-	// directory.
-	// Functions provided on FunctionPaths are globally scoped.
-	// If FunctionPaths length is > 0, then NoFunctionsFromInput defaults to true
-	FunctionPaths []string
+	// FunctionPath  allows functions to be specified by a path to function config
+	// file
+	FunctionPath string
```

Comment:
We should rename it to FnConfigPath ? and comment isn't accurate.

Reply:
renamed